.gitmodules

```diff
@@ -0,0 +1,20 @@
+[submodule "sorc/UFS_UTILS"]
+	path = sorc/UFS_UTILS
+	url = https://github.com/ufs-community/UFS_UTILS
+	branch = develop
```

Comment:
@MichaelLueken, the `branch` line may not be necessary, but it would be good to add `ignore = dirty` instead to exclude any change in the submodules.

Reply:
Thank you very much for the suggestion, @chan-hoo!  I will try removing the `branch` line and adding `ignore = dirty` instead.

Reply:
if this helps: In the UFSWM we keep the branch field. It's useful to know in a PR what submodules are bringing in updates and what branch the developer has those updates in as it's important for developers to make branches different names than develop. Maybe not all submodules use a "develop" branch.

Reply:
Thanks, @BrianCurtis-NOAA!  You make a very good point, if users are applying changes to a component, it would be nice to see the fork and branch that they are using for those changes.  There is no harm in keeping the `branch` line, so I will go ahead and keep it for now.